demo/main/osdet/osdet_demo_wt_got10k_image.py

```diff
@@ -134,4 +133,4 @@ def make_parser():
     cv2.imshow("im_concat", im_concat)
     cv2.waitKey(0)
 
-    # from IPython import embed;embed()
+    from IPython import embed;embed()

```

Comment:
can be removed

Reply:
This line of code is meant to store working environment (in Python) and leave users to check and explore themselves.